terraform/main.tf

```
@@ -181,6 +181,33 @@ module "grants_prepared_data_bucket" {
   ]
 }
 
+module "ffis_source_data_bucket" {
+  source  = "cloudposse/s3-bucket/aws"
+  version = "3.0.0"
+  context = module.s3_label.context
+  name    = "ffis_source_data"
```

A bit pedantic, but I think this should be called something like "email_delivery_bucket" (or simply, "emails"), for two reasons:
1. The "source data" bucket is ultimately what will be used for storage of `.eml` files from FFIS, as well as some of the derived raw data.
2. I see this bucket as being scoped more to SES email delivery in general – it's more serving as a holding ground for just-arrived email files that haven't yet been moved to the source data bucket for processing (or potentially never will because it's the sender is unrecognized).

```suggestion
module "email_delivery_bucket" {
  source  = "cloudposse/s3-bucket/aws"
  version = "3.0.0"
  context = module.s3_label.context
  name    = "email_delivery"
```

---

Yup, sounds good to me.  Changed in this latest commit.